images/plbase/plbase-install.sh

```diff
@@ -64,6 +64,13 @@ rm -rf /tmp/pgvector
 dnf -y remove postgresql15-server-devel
 dnf -y autoremove
 
+# Install d2
+echo "installing d2..."
+VERSION=v0.6.8 OS=linux ARCH=amd64 curl -fsSLO \
+    "https://github.com/terrastruct/d2/releases/download/$VERSION/d2-$VERSION-$OS-$ARCH.tar.gz" \
+    && tar -xzf "d2-$VERSION-$OS-$ARCH.tar.gz" \
+    && make -sC "d2-$VERSION" install PREFIX=/usr/local/bin
```

Comment:
IMO we shouldn't install this in the base image. I don't want this to be made available for use by instructors (yet).

Reply:
Does this go in the main PrairieLearn container?

Reply:
Yes: https://github.com/PrairieLearn/PrairieLearn/blob/b8cb11c28489bdad4b0e7d6b18ed6939442d4e93/Dockerfile#L3

Reply:
Moved over, but wasn't sure if you wanted it in `init.sh` or as a `RUN step`.

Reply:
Sorry if I wasn't clear: this shouldn't be installed in _any_ PL Docker image.

Reply:
So you want this as a CI step?

Reply:
Also if so, how should it pass the 'docker' checks?

Reply:
You already added it as a CI step in the `native-checks` job. That's the job that builds the docs to check for errors/warnings, so you should be good to go. `build-core-images` doesn't build the docs AFAIK, so it shouldn't be necessary there.